src/emulator/remoteconfig/server.ts

```diff
@@ -0,0 +1,78 @@
+import * as express from "express";
+import { RemoteConfigEmulator } from "./index";
+import * as cors from "cors";
+
+/**
+ * @param defaultProjectId
+ * @param emulator
+ */
+export function createApp(
+  defaultProjectId: string,
+  emulator: RemoteConfigEmulator
+): Promise<express.Express> {
+  const app = express();
+  app.use(cors("*" as any));
+
+  app.put("/revert", (req, res) => {
+    emulator.loadTemplate();
+    res.json(emulator.template);
+  });
+
+  // Please note you should set the Remote Config minimal fetch interval to 0, so it refreshes every time
+  // Otherwise caching will cause you headaches
+  app.post("/v1/projects/:projectId/namespaces/firebase:fetch", (req, res) => {
+    res.header("etag", `etag-${req.params["projectId"]}-firebase-fetch--${new Date().getTime()}`);
+    res.header(
+      "Access-Control-expose-Headers",
```

Comment:
Not sure if it has any effect, but i think this should be Expose-Headers 
(https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers)

Reply:
Done!